btr-web/btr-main-app/services/file-significant-individual.ts

```diff
@@ -109,6 +111,7 @@ const getPersonAndOwnershipAndControlStatements = (sif: SignificantIndividualFil
 }
 
 const convertToBtrBodsForSubmit = (sif: SignificantIndividualFilingI): BtrFilingI => {
+  // todo: fixme: with new changes
```

Comment:
Can probably remove this comment or is there work needed in here for a future ticket?